### DISCUSSION TITLE
Speaker Connections

### DISCUSSION THREAD

**Discussion by FA-Bubba** [SamDel/ChromeCast-Desktop-Audio-Streamer#111] [Q&A]
When I first learned about DAS, I was already searching on how to create wireless multi-room stereo, and had a few false starts...

I initially bought a bunch of the QFX E-350 Schubert Multi-Room Wifi & Bluetooth Speakers, which are now re-boxed because they default to Bluetooth, and must be manually switched to Wi-Fi each time they are restarted; also, they worked better with the QFX Tunes App, which must be run from an iPhone or iPad.  They worked, but quite a hassle to keep streaming 24x7...

At some point, I discovered Chromecast Audio, and just after I tested one, I learned that Google was discontinuing them (this was early 2019), so I bought as many as I could without paying highly inflated prices...  I really wanted Wi-Fi and NOT Bluetooth speakers, and Chromecast Audio (CCA) did that.

I eventually had 10 CCAs, and I also found stereo speaker pairs (Philips SPA4270BT/37) that accepted the 3mm connectors, so I acquired 10 of those, which ultimately enabled me to set up 10 locations for Wi-Fi stereo streaming.   Later, I added some Google Minis as part of my Google Home/Assistant setup, and added those speakers to DAS.  With some great advice from SamDel, I created some Speaker Groups in Google Assistant so the audio would be synchronized.

Long story short, I ended up with DAS casting to 14 sets of speakers.

I am now wondering what other speaker setups work well with DAS, because Chromecast Audio is no longer available (except for unreasonably high prices); likewise with the Philips SPA4270BT/37 speakers...

What types of speaker setups are others using for DAS to cast to?    ...I would like to tell my friends about DAS, but need some ideas for non-Chromecast Audio methods, so they can build a complete solution.
